### PR TITLE
[MINOR] Sort out the code logic of kyuubi/tpcds

### PR DESCRIPTION
**Pull request by zwangsheng**
zwangsheng [Closes #442](https://github.com/yaooqinn/kyuubi/pull/442)    Target Issue [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

There is a tpcds generate in kyuubi/dev/kyuubi-tpcds.
Helpe to sort out the code logic and modify the pom dependency of kyuubi-tpcds.
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/TableGenerator.scala [hunk 1]
```diff
@@ -85,20 +84,21 @@ case class TableGenerator(
...
         s" -DISTRIBUTIONS tpcds.idx" +
         s" -FORCE Y" +
         s" -QUIET Y"
 
       val builder = new ProcessBuilder(cmd.split(" "): _*)
       builder.directory(tempDir)
-      logger.info(s"Start $cmd at " + builder.directory())
       builder.redirectError(Redirect.INHERIT)
-      val data = Paths.get(tempDir.toString, s"${name}_${i}_$parallelism.dat")
+      logger.info(s"Start $cmd at " + builder.directory())
```
Comment (pan3793) [inline dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/TableGenerator.scala:96]: Can we use single style? i.e. replace `+` by `${}`

**Review by yaooqinn — APPROVED** [review]

File: dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/TableGenerator.scala (hunk 1 above)
Comment (zwangsheng) [reply dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/TableGenerator.scala:96]: ok, i will fix it

**Comment by yaooqinn** [comment]
Let's try to make our PR title and description more specific, please it is not obvious to the reviewers before they reach the code diff part

**Comment by zwangsheng** [comment]
> Let's try to make our PR title and description more specific, please it is not obvious to the reviewers before they reach the code diff part

ok, i will modify those

**Review by pan3793 — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.2.0
